react/Tooltip/Tooltip.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import ReactTooltip from 'react-tooltip'
+import styles from './tooltip.styl'
+//Only needed for Preact?
+React.Fragment = 'x-fragment'
+const Tooltip = ({ dataTip, dataFor, children }) => {
+  const newChild = React.cloneElement(children, {
+    dataTip,
+    dataFor
+  })
+  return (
+    <React.Fragment>
+      {newChild}
+      <ReactTooltip
+        place="bottom"
+        effect="solid"
+        className={styles['shared-tooltip']}
```

Comment:
I know I've to open the API to place / effect etc.

Reply:
I did not understand ^